### PR TITLE
[Merged by Bors] - Ssz state

### PR DESCRIPTION
**Pull request by paulhauner**
## Issue Addressed

NA

## Proposed Changes

Adds a `lighthouse/beacon/states/:state_id/ssz` endpoint to allow us to pull the genesis state from the API.

## Additional Info

NA

### CODE REVIEW

**Comment by paulhauner** [comment]
@realbigsean this is along the lines of what you're working on, do you mind giving it a review? :)

File: beacon_node/http_api/src/lib.rs [hunk 1]
```diff
@@ -1616,14 +1617,38 @@ pub fn serve<T: BeaconChainTypes>(
...
-        .and(chain_filter)
+        .and(chain_filter.clone())
         .and_then(|epoch: Epoch, chain: Arc<BeaconChain<T>>| {
             blocking_json_task(move || {
                 validator_inclusion::global_validator_inclusion_data(epoch, &chain)
                     .map(api_types::GenericResponse::from)
             })
         });
 
+    // GET lighthouse/beacon/state/{state_id}/ssz
```
Comment (realbigsean) [inline beacon_node/http_api/src/lib.rs:1628]:
```suggestion
    // GET lighthouse/beacon/states/{state_id}/ssz
```

File: book/src/api-lighthouse.md [hunk 2]
```diff
@@ -177,3 +177,13 @@ See [Validator Inclusion APIs](./validator-inclusion.md).
 ### `/lighthouse/validator_inclusion/{epoch}/global`
 
 See [Validator Inclusion APIs](./validator-inclusion.md).
+
+### `/lighthouse/beacon/states/{state_id}/ssz`
```
Comment (realbigsean) [inline book/src/api-lighthouse.md:181]: Adding the state_id options here might be good, or referencing the eth2 api. Might not be clear otherwise.

**Review by realbigsean — CHANGES_REQUESTED** [review]
Sorry it took me so long to review this! I missed it initially.  Left a few minor doc comments.

Another note: I don't think this'll work properly for genesis until #1783 is merged (I didn't test it out though)

**Comment by paulhauner** [comment]
Thanks! All review comments addressed, I'll go ahead and merge assuming I have approval (changes are trivial).

bors r+

**Comment by paulhauner** [comment]
bors retry

**Pull request closed without merging**